### DISCUSSION TITLE
Support for keyboard shortcuts (like VSCode)

### DISCUSSION THREAD

**Discussion by reginoxcf** [cpeditor/cpeditor#1180] [Ideas]
Can we have some support for keyboard shortcuts for actions like:
- Duplicate line
- Delete line
- ...
It would be very helpful, thanks!

**Comment by ouuan** [comment]
Duplicate line: <kbd>Ctrl+Shift+D</kbd>

Delete line: <kbd>Ctrl+Shift+K</kbd>

Customizable: #214
